src/core/defs.jl

```diff
@@ -132,17 +132,18 @@ description(def::DatumDef) = def.description
 
 unit(def::DatumDef) = def.unit
 
-# step_size(md::ModelDef) = step_size(indexvalues(md))
+function start_step(md::ModelDef)
```

Comment:
I find the term `start` in `start_step` a bit confusing, since it refers both to the "first" timestep "start" (time label) value. I think it would be clearer to rename `start_step` => `first_and_step` since it returns both the first value and the first step?

Also, reduce redundancy by having first of these methods call the second (short funcs get inlined anyway): 
```
function first_and_step(md::ModelDef)
    keys::Vector{Int} = timelabels(md) # keys are the start_times of the model runs
    return first_and_step(keys)
end

function first_and_step(values::Vector{Int})
     return values[1], (length(values) > 1 ? values[2] - values[1] : 1)
end
```

Assuming `Vector{Int}` is fine for this first pass, but thinking about non-Int time values, we could remove the Int assumptions but parameterize the second func for efficiency:
```
function first_and_step(md::ModelDef)
    labels = timelabels(md)       # labels are the start_times of the model runs
    return first_and_step(labels)
end

function first_and_step(values::Vector{T}) where T
     return values[1], (length(values) > 1 ? values[2] - values[1] : T(1))   # I'm not sure this works for time values...
end
```